TestData/Insomnia_2024-10-15.json

```diff
@@ -0,0 +1 @@
es":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"req_4ee9b2cc70764e5f8faab8feb2a987db","parentId":"fld_498a9ba06737490a8eeb97510ffd073f","modified":1728407736030,"created":1726152116657,"url":"http://localhost:6070/api/GetEpisode?EpisodeId=245395","name":"GetEpisode EpisodeDataService 245395","description":"","method":"GET","body":{},"parameters":[],"headers":[{"name":"User-Agent","value":"insomnia/9.3.3"}],"authentication":{},"metaSortKey":-1728058041472,"isPrivate":false,"pathParameters":[],"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"req_c69ade112e28480fb68d6496544d2480","parentId":"fld_498a9ba06737490a8eeb97510ffd073f","modified":1728915187959,"created":1728294743548,"url":"http://localhost:6070/api/GetEpisode?EpisodeId=656047","name":"GetEpisode EpisodeDataService 656047","description":"","method":"GET","body":{},"parameters":[],"headers":[{"name":"User-Agent","value":"insomnia/9.3.3"}],"authentication":{},"metaSortKey":-1728058041422,"isPrivate":false,"pathParameters":[],"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"req_550e367c5c34409fba867ebc8a4413fe","parentId":"fld_498a9ba06737490a8eeb97510ffd073f","modified":1728915190307,"created":1726152116655,"url":"http://localhost:6007/api/CreateEpisode","name":"CreateEpisode EpisodeDataService","description":"","method":"POST","body":{"mimeType":"application/json","text":"{\n\t\"EpisodeId\": \"656047\",\n\t\"ParticipantId\": \"123123123\"\n}"},"parameters":[],"headers":[{"name":"Content-Type","value":"application/json"},{"name":"User-Agent","value":"insomnia/9.3.3"}],"authentication":{},"metaSortKey":-1728058041372,"isPrivate":false,"pathParameters":[],"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"env_1b4aef4b7e814314a19cff287fe1eb62","parentId":"wrk_15988441697c4b84b92bb74791a24fcd","modified":1726152116651,"created":1726152116651,"name":"Base Environment","data":{},"dataPropertyOrder":null,"color":null,"isPrivate":false,"metaSortKey":1722249723675,"_type":"environment"},{"_id":"jar_7c7f33006f464afa8ae78b213a1d1956","parentId":"wrk_15988441697c4b84b92bb74791a24fcd","modified":1726152116652,"created":1726152116652,"name":"Default Jar","cookies":[],"_type":"cookie_jar"},{"_id":"spc_515fbcc705bd4b64bbf6bad5678afcb1","parentId":"wrk_15988441697c4b84b92bb74791a24fcd","modified":1726152116660,"created":1726152116660,"fileName":"Service Insights","contents":"","contentType":"yaml","_type":"api_spec"}]}
```

Comment:
Still a few references to test-happy-path.json in here

Reply:
will check for references and remove or update to the CSV

Reply:
new insomnia file uploaded without the happy-path JSON